xm-commons-security/src/main/java/com/icthh/xm/commons/security/XmAuthenticationContext.java

```diff
@@ -130,4 +135,13 @@ public interface XmAuthenticationContext {
      */
     String getAdditionalDetailsValue(String key, String defaultValue);
 
+    String getClientId();
```

Comment:
What about to add getUsername() method?,- similarly to getClientId()

Reply:
@pkev777 
regarding getUsername() we have `getLogin()` and `getRequiredLogin()`

Reply:
@pkev777  Added getRequiredClientId() method and getClientId now return Optional